src/App.tsx

```diff
@@ -124,6 +124,7 @@ const App = (): JSX.Element => {
       );
       dispatch(isPolkaBtcLoaded(true));
       setIsLoading(false);
+
       // NOTE: Catch clause variable type annotation must be 'any' or 'unknown'. Use of any here
```

Comment:
This one's one on me, but this comment can be taken out as it's not relevant to the TS version we're running on the project (it was introduced in a later version). We'll address this again when we update 
TypeScript.

Reply:
@nud3l 
Please make sure that you're using TypeScript version installed in the node_modules rather than in VSCode.
The other day I discussed with Tom on this issue. The latest TypeScript could introduce new error cases.

Reply:
Good point, thanks!